docs/pages/workflows/control-flow.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Control flow
+description: Learn how to control the flow of Workflows with conditions that run certain jobs based on the result of previous jobs.
+---
+
+You can control the flow of your workflows with conditions that run certain jobs based on the result of previous jobs.
+
+## `needs`
+
+You can add an list of job names to the `needs` keyword on a job. If a job in the list fails, the job will be skipped.
```

Comment:
```suggestion
You can add a list of previous job names using the `needs` keyword on a job. If any job from that list fails, the current job will be skipped.
```

At first it was a bit confusing to read this sentence to quickly understand what was happening. In the last sentence, I am not sure if the current job is skipped or the job failed from the list is skipped.

Assuming that the only previous job names can be added to the job using the `needs` keyword, let's explicitly mention that to improve clarity.

Please correct me if my assumption is wrong and feel free to update the sentence accordingly :)